pkg/diagnose/deployments.go

```diff
@@ -200,8 +201,15 @@ func checkPodsStatus(k8sClient kubernetes.Interface, namespace string, status re
 		return
 	}
 
+	numOfActiveGws := 0
+
 	for i := range pods.Items {
 		pod := &pods.Items[i]
+
+		if (pod.Labels["app"] == names.GatewayComponent) && (pod.Labels["gateway.submariner.io/status"] == "active") {
```

Comment:
We could the status constant here

```suggestion
		if (pod.Labels["app"] == names.GatewayComponent) && (pod.Labels["gateway.submariner.io/status"] == submarinerv1.HAStatusActive) {
```